# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/nextpnr/hotfix-no-tests.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- nextpnr-nextpnr-0.8/CMakeLists.txt.vanilla	2025-04-15 12:31:55.811844681 +0200
+++ nextpnr-nextpnr-0.8/CMakeLists.txt	2025-04-15 12:32:42.423344399 +0200
@@ -239,7 +239,9 @@
 add_subdirectory(json)
 add_subdirectory(rust)
 
+if (BUILD_TESTS)
 add_subdirectory(tests/gui)
+endif()
 
 add_custom_target(nextpnr-all-bba)
 
